polaris-react/src/components/Tabs/Tabs.scss

```diff
@@ -149,13 +145,9 @@
     }
 
     &:not([aria-disabled='true']):active {
-      background-color: var(--p-color-bg-subdued);
+      background-color: var(--p-color-bg-secondary-experimental);
       color: var(--p-color-text-primary);
       z-index: var(--p-z-index-1);
-
-      #{$se23} & {
```

Comment:
Same as **[this comment](https://github.com/Shopify/polaris/pull/9619/files#r1253604029)**, see: 

https://github.com/Shopify/polaris/blob/5e2c3c532a9a0ee7aa6fd49439be118364a44c0d/polaris-react/src/components/Tabs/Tabs.scss#L123